test/check-application

```diff
@@ -3005,6 +3026,65 @@ class TestApplication(testlib.MachineCase):
         else:
             self.assertNotIn("Podman", b.text("#host-apps"))
 
+    @testlib.skipImage("nginx not installed", "centos-*", "rhel-*", "debian-*", "ubuntu-*", "arch")
+    @testlib.skipOstree("nginx not installed")
+    def testNonSearchableRegistry(self):
+        b = self.browser
+        m = self.machine
+
+        self.setupRegistry()
+
+        # Nginx config simulates behavior of ghcr.io in terms of `podman search` and `podman manifest inspect`
+        self.execute(True, f"""
+            echo '{NGINX_DEFAULT_CONF}' > /etc/nginx/conf.d/default.conf
+            setsebool -P httpd_can_network_connect 1
+            systemctl start nginx.service
+        """)
+        self.addCleanup(m.execute, """
+            systemctl stop nginx.service
+            setsebool -P httpd_can_network_connect 0
+            rm -f /etc/nginx/default.conf
```

Comment:
Asymmetric -- this isn't the file that the code above created. Please use `self.write_file()` :grin: